src/LineItem/Interfaces/LineItemRequestOptions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+
+namespace Skarp.HubSpotClient.LineItem
+{
+    public class LineItemListRequestOptions
+    {
+        /// <summary>
+        /// Get or set the continuation offset when calling list many times to enumerate all your line items.
+        /// The API will return up to 100 results per page.
+        /// </summary>
+        /// <remarks>
+        /// The return DTO from List contains the current "offset" that you can inject into your next list call 
+        /// to continue the listing process
+        /// </remarks>
+        public long? Offset { get; set; } = null;
+        public List<string> PropertiesToInclude { get; set; } = new List<string>();
+    }
+
+    public class LineItemGetRequestOptions
```

Comment:
**nitpick(non-blocking)** : I prefer one file one class but in this instance I can make an exception.

Reply:
Yeah I agree. Actually I wasn’t sure, but I noticed request options combined in one other place too so I assumed it as the standard.

Reply:
perhaps I have missed that in a previous PR or just got lazy 👀  Candidate for refactor at a later point.